src/utils/utils_load_library.h

```diff
@@ -39,11 +39,15 @@ static inline void *util_open_library(const char *filename) {
 static inline int util_close_library(void *handle) {
     // If the FreeLibrary function succeeds, the return value is nonzero.
     // If the FreeLibrary function fails, the return value is zero.
-    return (FreeLibrary(handle) == 0);
+    return (FreeLibrary((HMODULE)handle) == 0);
 }
 
 static inline void *util_get_symbol_addr(void *handle, const char *symbol) {
-    return GetProcAddress(handle, symbol);
+    if (handle == NULL) {
+        // handle can't be null
+        handle = GetModuleHandle("ze_loader.dll");
```

Comment:
this can't be hardcoded - it should be the parameter to this function. We should either uses provided handle OR dll name

Reply:
I've changed it